hr_attendance_management/static/src/js/attendance.js

```diff
@@ -36,27 +35,27 @@ odoo.define('hr_attendance_management.attendance', function (require) {
                     });
 
                     // auto-counter
-                    $('#moment_pl').html(Date.now());
                     setInterval(function () {
+                        $('#moment_pl').html(Date.now());
```

Comment:
I'm not sure, but is the timer correctly increased (meaning the worked hours are additioning if we stay on the page?)